scripts/dev/e2e.py

```diff
@@ -255,6 +255,12 @@ def parse_args() -> argparse.Namespace:
         help="Watch all namespaces",
         action="store_true",
     )
+    parser.add_argument(
+        "--distro",
+        help="The distro of images that should be used",
+        type=str,
```

Comment:
[nit] Can we use `choices` here to make sure that we don't pass something completely different? https://docs.python.org/3/library/argparse.html#choices